styleguide/derek/_partials/solutions/testimonials.ejs

```diff
@@ -1,15 +1,16 @@
 <!-- Testimonials -->
-<div class="bg-blue-gray standard-padding">
-  <div class="container">
+<div class="quote">
+  <div class="quote-container">
```

Comment:
To keep consistent with changes @writes made on other patterns, can use `quote-wrapper` instead of `quote-container`?  Container has a very strict meaning in bootstrap, and while this technically applies in this case, we wan't to avoid any ambiguity.  I realize there's a `quote-wrapper` below, so that would probably need to be named something else.  Perhaps `quote-textWrapper` or something like that.